src/batch-client/index.ts

```diff
@@ -384,6 +385,19 @@ export class ZimbraBatchClient {
 			body: denormalize(CreateSignatureRequest)(options)
 		});
 
+	public createTag = (_options: CreateTagOptions) => {
+		const { name, color } = _options;
+		return this.jsonRequest({
+			name: 'CreateTag',
+			body: {
+				tag: {
+					name,
```

Comment:
do we need destructuring and separate keys here?
```
tag: {
    ...options
}
```

Reply:
Done